<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guiClass</class>
 <widget class="QWidget" name="guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="eqList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="aLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="bLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="cLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="sol1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sol2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="computeButton">
       <property name="text">
        <string>Compute Solutions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
